cves/CVE-2018-14394.yml

```diff
@@ -94,18 +97,8 @@ fixes:
 - commit: ''
   note: ''
 vccs:
-- commit: 3664703780a3d882937c131e4104f8d21e71b738
-  note: Identified by archeogit,
-- commit: e45ccf79be6f157a190c5c9c58e246b5b03c3510
-  note: Identified by archeogit,
-- commit: 5cb49ca11bca4bedb71ee0204e8115fa970d6dbb
-  note: Identified by archeogit,
-- commit: 6f69f7a8bf6a0d013985578df2ef42ee6b1c7994
-  note: Identified by archeogit,
-- commit: b7d3dd907f51c1b86c5aad7e2ec4870da53e095e
-  note: Identified by archeogit,
-- commit: 5616f85deb729d181e2a923b2c74371fae6e8398
-  note: Identified by archeogit,
+- commit: f578f93838bad57be9f4e476926c015a72b0467b
+  note: There were lots of refactoring of variable names, so this was a bit hard to find
```

Comment:
The time gap between the VCC and the fix is huge. Can you add some more on what changed in the code/code around it/file during this time (lots of new additions)?